docs/entities/OfferReview.md

```diff
@@ -0,0 +1,24 @@
+# OfferReview
+
+## Descripción
+Representación de una resenya en la plataforma.
+
+## Propiedades
+* id: Identificador único de la reseña.
+* buyerId: Identificador único del comprador.
+* userImageUrl: URL de la imagen del usuario que escribió la reseña.
+* content: Texto de la reseña, que describe la experiencia del comprador con la oferta.
+* offerTitle: Título de la oferta sobre la que se realiza la reseña.
+* date: Fecha en la que se escribió la reseña.
```

Comment:
I'd use the standard "createdAt" here. If we're going to allow users to edit reviews I'd also add an "updatedAt" field. This is very standard, as most databases can include these automatically as audit fields.